todoapp/projects/models.py

```
@@ -1,18 +1,37 @@
-        Add string representation for this model with project name.
+    Add string representation for this model with project name.
     """
 
+    name = models.CharField(max_length=100)
+    members = models.ManyToManyField(
+        settings.AUTH_USER_MODEL,
+        through="ProjectMember",
+    )
+    max_members = models.PositiveIntegerField(null=True)
+    status = models.PositiveSmallIntegerField(
+        choices=Status.choices, default=Status.TO_BE_STARTED
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)